src/cascadia/TerminalControl/SearchBoxControl.xaml

```diff
@@ -0,0 +1,122 @@
+﻿<UserControl

```

Comment:
Thank you for putting automation properties in here!!! 😊

@leonMSFT since you're working on the tab switcher, the automation properties used here are something you'd have to add in. This is a good example of how to do that.